scipy/interpolate/_interpolate.py

```diff
@@ -2353,22 +2353,24 @@ class RegularGridInterpolator:
     Interpolation on a regular grid in arbitrary dimensions
 
     The data must be defined on a regular grid; the grid spacing however may be
-    uneven. Linear and nearest-neighbor interpolation are supported. After
-    setting up the interpolator object, the interpolation method (*linear* or
-    *nearest*) may be chosen at each evaluation.
+    uneven. Linear, nearest-neighbor, spline interpolations are supported.
+    After setting up the interpolator object, the interpolation method may be
+    chosen at each evaluation.
 
     Parameters
     ----------
     points : tuple of ndarray of float, with shapes (m1, ), ..., (mn, )
         The points defining the regular grid in n dimensions.
 
     values : array_like, shape (m1, ..., mn, ...)
-        The data on the regular grid in n dimensions.
+        The data on the regular grid in n dimensions. Complex data can be
+        acceptable.
```

Comment:
I enhanced the doc for `RegularGridInterpolator` to clarify complex data is acceptable because some people might don't know it in #2650